packages/account/src/Components/api-token/api-token.scss

```diff
@@ -72,22 +73,44 @@
                 border-bottom: 2px solid var(--general-section-1);
             }
 
+            white-space: nowrap;
             padding: 2.4rem 0;
             text-align: left;
         }
         &-cell {
+            vertical-align: middle;
             &-row {
                 border-bottom: 1px solid var(--general-section-1);
             }
 
-            padding: 2.4rem 0;
+            padding: 1.6rem 0;
```

Comment:
In the design the last cell(delete button) has padding-right
Design:
![image](https://user-images.githubusercontent.com/82078941/176376410-7dd0d577-f5af-41ca-be65-5b4483930bed.png)

Test link:
![image](https://user-images.githubusercontent.com/82078941/176376289-6a446105-a00d-4102-aa59-f9bf77c706ab.png)